common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
+                "items": [
+                    {
+                        "short_description": _("Interleave instructional content and exercises"),
+                        "long_description": _(
+                            "Ensure that you design your course so that instructional content (videos, text, etc) are"
+                            "interspersed with exercises."
                         ),
                         "is_checked": False,
                         "action_url": "CourseOutline",
-                        "action_text": _("Edit Course Outline"),
+                        "action_text": _("View Course Outline"),
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Designate a Subsection as Graded"),
+                        "short_description": _("Clearly articulated assignments"),
                         "long_description": _(
-                            "Set a Subsection to be graded as a specific assignment type. Assignments within graded "
-                            "Subsections count toward a student's final grade."
+                            "Ensure that your assignment questions and prompts are written clearly and articulately."
                         ),
                         "is_checked": False,
                         "action_url": "CourseOutline",
-                        "action_text": _("Edit Course Outline"),
+                        "action_text": _("View Course Outline"),
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Reordering Course Content"),
-                        "long_description": _("Use drag and drop to reorder the content in your course."),
+                        "short_description": _("Clearly communicated deadlines"),
+                        "long_description": _(
+                            "Ensure that the deadline for every assignment is clearly communicated in the assignment itself."
+                        ),
                         "is_checked": False,
                         "action_url": "CourseOutline",
-                        "action_text": _("Edit Course Outline"),
+                        "action_text": _("View Course Outline"),
                         "action_external": False,
                     },
+                ],
+            },
+            {
+                "short_description": _("Course Videos"),
+                "items": [
                     {
-                        "short_description": _("Renaming Sections"),
-                        "long_description": _("Rename Sections by clicking the Section name from the Course Outline."),
+                        "short_description": _("Include videos"),
+                        "long_description": _(
+                            "Ensure that your course includes video content"
```

Comment:
Verify that your course includes original videos that are hosted in the edX video player.